clang/include/clang/Driver/Options.td

```diff
@@ -3389,10 +3389,12 @@ def mcmodel_EQ_medany : Flag<["-"], "mcmodel=medany">, Group<m_riscv_Features_Gr
   HelpText<"Equivalent to -mcmodel=medium, compatible with RISC-V gcc.">;
 def menable_experimental_extensions : Flag<["-"], "menable-experimental-extensions">, Group<m_Group>,
   HelpText<"Enable use of experimental RISC-V extensions.">;
-def mxcheri_rvc : Flag<["-"], "mxcheri-rvc">, Group<m_riscv_Features_Group>,
+// TODO: Remove these options (https://github.com/CTSRD-CHERI/llvm-project/issues/681)
+def mxcheri_rvc : Flag<["-"], "mxcheri-rvc">, Group<clang_ignored_m_Group>,
```

Comment:
This won't re-enable it after -mxcheri-norvc? I'd just leave things alone and flip the default by itself...

Reply:
Good point, I can just add a new "-mno-xcheri-norvc" and make this an alias so that the shared logic works as expected.